src/ynEIGEN/LSDRateProvider.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: BSD 3-Clause License
+pragma solidity ^0.8.24;
+
+import {Initializable} from "lib/openzeppelin-contracts-upgradeable/contracts/proxy/utils/Initializable.sol";
+import {IERC20} from "lib/openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
+import {IERC4626} from "lib/openzeppelin-contracts/contracts/interfaces/IERC4626.sol";
+import {IstETH} from "src/external/lido/IstETH.sol";
+import {IrETH} from "src/external/rocketpool/IrETH.sol";
+import {IswETH} from "src/external/swell/IswETH.sol";
+
+
+contract LSDRateProvider is Initializable {
```

Comment:
The contract is never initialized, inheritance of `Initializable` can be removed.

Reply:
fixed in base branch